notebooks/110-ct-segmentation-quantize/110-ct-segmentation-quantize-nncf.ipynb

```diff
@@ -477,22 +481,24 @@
     "    return images\n",
     "\n",
     "\n",
-    "data_loader = torch.utils.data.DataLoader(dataset, batch_size=4)\n",
+    "data_loader = torch.utils.data.DataLoader(dataset)\n",
     "calibration_dataset = nncf.Dataset(data_loader, transform_fn)\n",
     "quantized_model = nncf.quantize(\n",
     "    model,\n",
     "    calibration_dataset,\n",
-    "    target_device=nncf.TargetDevice.CPU,\n",
     "    # Do not quantize LeakyReLU activations to allow the INT8 model to run on Intel GPU\n",
-    "    ignored_scope=nncf.IgnoredScope(patterns=[\"{re}.*LeakyReLU*\"])\n",
+    "    ignored_scope=nncf.IgnoredScope(patterns=[\".*LeakyReLU.*\"])\n",
```

Comment:
@aleksandr-mokrov these changes in ignored scope are expected or made accidentally?

Reply:
They are expected, from @alexsu52